### PR TITLE
Adds a UserVoice link to known issues section

### PR DESCRIPTION
**Pull request by SQLGB**
For 18.4, adds a UserVoice link to known issues section which exists for 18.3.1

### CODE REVIEW

**Comment by PRMerger17** [comment]
@SQLGB : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/ssms/release-notes-ssms.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ SSMS 18.4 is the latest general availability (GA) release of SSMS. If you need a
 
 * Database Diagram created from SSMS running on machine A cannot be modified from machine B (SSMS crashes). See [UserVoice](https://feedback.azure.com/forums/908035/suggestions/37992649) for more details.
 
-* There are redraw issues when switching between multiple query windows. See UserVoice for more details. A workaround for this issue is to disable hardware acceleration under *Tools > Options*.
+* There are redraw issues when switching between multiple query windows. See [UserVoice](https://feedback.azure.com/forums/908035/suggestions/37474042) for more details. A workaround for this issue is to disable hardware acceleration under *Tools > Options*.
```
Comment (markingmyname) [inline docs/ssms/release-notes-ssms.md]: Please change the link to - https://feedback.azure.com/forums/908035-sql-server.

Comment (markingmyname) [reply docs/ssms/release-notes-ssms.md]: @SQLGB

**Comment by markingmyname** [comment]
Reviewed link and it is correct.  approved.

**Pull request merged**